object-manipulation-1/src/get-value.js

```diff
@@ -0,0 +1,6 @@
+/* exported getValue */
+function getValue(object, key) {
+
+  return (object[key]);
```

Comment:
Parentheses are unnecessary here. You only need parentheses around an expression when you need to adjust the order of operations